css/components/Card.css

```diff
@@ -0,0 +1,159 @@
+@import "../style.css";
+
+body{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+/* Card with Badge */
+.badge, .dismiss, .text-overlay, .text, .horizontal, .shadow{
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 200px;
+    height: 270px;
+    border: 1px solid var(--badge-offline);
+    background-color: var(--badge-offline);
+    border-radius: 1rem;
+    margin: .5rem;
+}
+
+.badge .far, .dismiss .far, .text-overlay .far, .text .far, .horizontal .far, .shadow .far{
+    position: absolute;
+    font-size: 1rem;
+    top: 9px;
+    right: 9px;
+}
+
+.badge-header, .dismiss-header, .text-overlay-header, .text-header, .horizontal-header, .shadow-header{
+    padding: 0 5px;
+    color: var(--default);
+    font-weight: bolder;
+}
+
+.badge-img, .dismiss-img, .text-overlay-img, .text-img, .horizontal-img, .shadow-img{
+    width: 100%;
+    height: 35%;
+}
+
+.badge-text, .dismiss-text, .text-overlay-text, .text-text, .horizontal-text, .shadow-text{
+    padding: 0 2px;
+}
+
+.badge-footer, .dismiss-footer, .text-overlay-footer, .text-footer, .horizontal-footer, .shadow-footer{
+    display: flex;
+    justify-content: space-around;
+}
+
+.btn{
+    background-color: var(--badge-offline);
+    border: none;
+
+}
+
+.primary a,.secondary a{
+    text-decoration: none;
+    color: var(--default);
+}
+
+.primary a:hover,.secondary a:hover{
+    color: var(--alert-warning);
+}
+
+.icon{
+    align-self: flex-end;
+    font-size: 1rem;
+    margin: 0 5px;
+    color: var(--default);
+}
+
+.fa-heart:hover{
+    color: var(--badge-dnd);
+}
+
+.fa-shopping-cart:hover{
+    color: var(--badge-online);
+}
+
+/* Card with dismiss */
+.dismiss .far{
+    display: none;
+}
+
+/* Card with text overlay */
+
+.text-overlay .far{
+    position: absolute;
+    color: var(--white-section);
+}
+
+.text-overlay-header h2{
+    position: absolute;
+    top: 6rem;
+    color: var(--white-section);
+}
+
+.text-overlay-header h3{
+    position: absolute;
+    top: 7.5rem;
+    color: var(--white-section);
+}
+
+.text-overlay-img{
+    height: 60%;
+    border-radius: 1rem 1rem 0 0;
+}
+
+/* Text only card */
+
+.text{
+    height: 150px;
+}
+
+.text-img{
+    display: none;
+}
+
+/* Horizontal Card */
+
+.horizontal{
+    height: 150px;
+}
+
+.horizontal-div{
+    height: 60%;
+    display: flex;
+}
+
+.horizontal .far{
+    display: none;
+}
+
+.horizontal-header{
+    order: 2;
+}
+
+.horizontal-header h2, .horizontal-header h3{
```

Comment:
It can better if givenc class names to element h2 and h3.

Reply:
class name updated

Reply:
class name updated